mozilla/fx-private-relay
# Add new user research recruitment banner

# New feature description

Closes MPP-2034, by re-enabling the interview recruitment banner with a new text and a new link.

Note that I also extracted all messages at the top (interview recruitment, CSAT survey, NPS survey) into a separate folder, so that results in some extra noise. You can probably just skip that commit (4aa50c23875412fc9a36163b3560dacefcd93287) - the primary one is a0bedb6f88b70a819d8c9e831768a0127b3b733c.

Compared to the previous interview recruitment survey, what's new is that this one is available in more countries than the US, and is only visible on the dashboard.

I also snuck in d993397277d3f169feb95697f7d77dfe9d577bee, which marks the API key field as read-only, which I'd apparently forgotten earlier. That caused the following error in tests:

```
Warning: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.
```

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/171620375-342b2dfa-1627-4699-8807-7a438a5f9650.png)

# How to test

Make sure your locale is set to one of the supported countries, with English as the language (e.g. `en-US`) and that you're logged in, then visit the dashboard. You should see the message.

# Checklist

- [x] l10n dependencies have been merged, if any. - None, English only.
- [x] All acceptance criteria are met. (Some unclarity about the required countries, asked for clarification in the ticket.)
